.github/workflows/playwright.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  schedule:
+    - cron: '0 8 * * *'
```

Comment:
So every day at 8AM (presumably in whatever timezone the GitHub Actions servers are running)?

Reply:
I think once a day is plenty. I imagine the risk of stage changing without going through several PRs (which already run the tests) is pretty low.

Reply:
So I don't think this is right.. Right now this `yml` is running e2e for smoke tests. We'd probably want to leave this alone.

The cron is supposed to be for the entire suite

Reply:
your're correct joe, we have another ticket that will split this up to its own yml, as we don't need to do the building etc, as we'll use stage/prod env anyway